src/static/webpack/rules/cssLoader.js

```diff
@@ -11,10 +11,15 @@ export default function ({ config, stage }) {
         'style-loader',
         {
           loader: 'css-loader',
-          options: {
-            sourceMap: true,
-            importLoaders: 1,
-          },
+          options: config.cssModules
+            ? {
+              importLoaders: 1,
+              modules: true,
+              localIdentName: '[name]__[local]___[hash:base64:5]',
+            } : {
+              sourceMap: true,
+              importLoaders: 1,
+            },
         },
```

Comment:
I would rather not encourage changes to the base loaders, and instead have this done through `config.webpack`. Thoughts?